bottomless2/src/lib.rs

```diff
@@ -0,0 +1,179 @@
+#![allow(dead_code, unused_variables, async_fn_in_trait)]
```

Comment:
whats up with the async fn in trait stuff? Probably complaining about the send issues?

Reply:
linter complains because the trait is public API